vispy/visuals/image.py

```diff
@@ -171,7 +518,28 @@ class ImageVisual(Visual):
             * 'hanning', 'hamming', 'hermite', 'kaiser', 'quadric', 'bicubic',
                 'catrom', 'mitchell', 'spline16', 'spline36', 'gaussian',
                 'bessel', 'sinc', 'lanczos', 'blackman'
-
+    texture_format: numpy.dtype | str | None
```

Comment:
It would be good to have examples showing the different behaviour, maybe a dedicated gallery section? Should be worked on after discussion of details.

Reply:
What do you think the example could show? I could make an example (or update an existing one) that shows the same image with the old behavior and one with a floating point texture. :thinking: Although the main benefit of this PR is the performance. Otherwise, we'd be trying to show the quantization/precision loss of the old behavior which may not be that great of an example.

Reply:
Oh, maybe gallery is not the right place. What I had in mind was kind of how-to example.

Reply:
Example in `<vispy root>/examples/` or in the docstring itself? Or somewhere else? Maybe it comes in a future PR?

Reply:
Yeah, sure, future PR! That's what I meant with "after discussion of details". Sorry for not being clear enough. And it would well fit in examples-folder.